free-programming-books.md

```diff
@@ -394,6 +399,7 @@
 
 * [Computer Science I - Draft](https://bitbucket.org/chrisbourke/computersciencei/src) - Dr. Chris Bourke (PDF) (:construction: *in process*)
 * [Computer Science II - Draft](http://cse.unl.edu/~cbourke/ComputerScienceTwo.pdf) - Dr. Chris Bourke (PDF) (:construction: *in process*)
+* [CS For All](https://www.cs.hmc.edu/csforall/)
```

Comment:
this is just the problem sets from a book. should we keep it?

Reply:
Good point, I missed that. Looks like there's a link to the actual book that costs almost $50 so I think it's definitely worth removing. Added a commit.